redash/query_runner/google_analytics.py

```diff
@@ -164,7 +164,7 @@ def run_query(self, query, user):
             params = json_loads(query)
         except:
             params = parse_qs(urlparse(query).query, keep_blank_values=True)
-            for key in params.keys():
+            for key in [*params]:
```

Comment:
```suggestion
            # Because we change keys list on line 170, we need to create a copy of the keys
            for key in [*params]:
```

Reply:
Another option that would make fiddling with iterators (and the need to explain it) redundant is:

```python
 query_string = parse_qs(urlparse(query).query, keep_blank_values=True) 
 params = {k.replace('-', '_'): ",".join(v) for k,v in query_string.items() }
```
We'll still have to explain `",".join`, but maybe we don't have to.

Reply:
I like this :+1: